packages/vite-plugin/package.json

```diff
@@ -15,7 +15,8 @@
   "author": "SEEK",
   "license": "MIT",
   "dependencies": {
-    "@vanilla-extract/integration": "^1.4.2"
+    "@vanilla-extract/integration": "^1.4.2",
+    "outdent": "^0.8.0"
```

Comment:
Realised we're already using `dedent` in the css package so we should probably be consistent.